applications/RomApplication/tests/structural_static_test_files/MainKratosHROM.py

```diff
@@ -8,10 +6,10 @@
 class TestStructuralMechanicsStaticHROM(StructuralMechanicsAnalysisROM):
 
     def __init__(self,model,project_parameters):
-        super(TestStructuralMechanicsStaticHROM,self).__init__(model,project_parameters)
+        super().__init__(model,project_parameters)
 
     def ModifyInitialGeometry(self):
-        super(TestStructuralMechanicsStaticHROM,self).ModifyInitialGeometry()
+        super().ModifyInitialGeometry()
```

Comment:
Shouldn't this be `ModifyAfterSolverInitialize` after what we spoke before?

Reply:
Indeed, I have changed it as well now